dockers/docker-router-advertiser/radvd.conf.j2

```diff
@@ -23,15 +34,15 @@ interface {{ name }}
     AdvOtherConfigFlag off;
     AdvLinkMTU 9100;
     AdvHomeAgentFlag off;
+{% for prefix in prefixes %}
```

Comment:
do we have some unit tests for this?

Reply:
not yet. Is there existing unit test for radv? Should it be added to https://github.com/Azure/sonic-buildimage/tree/master/src/sonic-config-engine/tests/sample_output ?

Reply:
yes, lets try add it there